### DISCUSSION TITLE
_dict file not created?

### DISCUSSION THREAD

**Discussion by kadiera** [espeak-ng/espeak-ng#1618] [Q&A]
Created all my files, theoretically following the examples, to add Lakota. lkt_dict isn't created. 

I'm wondering - there's no folder even close to the language family, so I added my own. Is there a reason to think this is why it didn't work?

**Comment by jaacoppi** [comment]
Can you publish your code & version history so we can take a look?

**Reply by kadiera** [reply]
https://github.com/kadiera/espeak-ng.git

**Comment by jaacoppi** [comment]
> https://github.com/kadiera/espeak-ng.git  

Your phsource/ph_lakota is empty. Fix that and see if helps.
